platform-core-module/src/main/java/com/softicar/platform/core/module/user/navigation/RecursivelyCollapseFoldersInput.java

```diff
@@ -1,23 +1,22 @@
 package com.softicar.platform.core.module.user.navigation;
 
-import com.softicar.platform.core.module.user.AGUser;
+import com.softicar.platform.core.module.user.UserPreferences;
 import com.softicar.platform.emf.attribute.field.bool.EmfBooleanInput;
 
 public class RecursivelyCollapseFoldersInput extends EmfBooleanInput {
```

Comment:
should be deleted

Reply:
Definitely. This was orphaned, and an oversight.